index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<div class="container">
+  <html>
+  <head>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.1/css/bootstrap.min.css" integrity="sha384-WskhaSGFgHYWDcbwN70/dfYBj47jz9qbsMId/iRN3ewGhXQFZCSftd1LZCfmhktB" crossorigin="anonymous">
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.1.1/js/bootstrap.min.js" integrity="sha384-smHYKdLADwkXOn1EmN1qk/HfnUcbVRZyYmZ4qpPea6sjB/pTJ0euyQp0Mk8ck+5T" crossorigin="anonymous"></script>    
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Excercise 1</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <style>
+    div{
+      padding:10px;
+    }
+  </style>
+</head>
+<body>
+  <div>
+    <img src ="http://www.girldevelopitfargo.com/images/GDI.jpg">
+    <h1>About</h1>
+
+    <div class="table-responsive">
+      <table class="table">
+        It can be intimidating for women to learn and ask questions when they are in the <strong>extreme minority</strong>strong.
+        While open and welcoming, today's budding developer community is up to <strong>91% male</strong>. If we can empower more females with the confidence in their technological capabilities <em>we can begin to change this landscape</em>.
+        
+      </table>
+    </div>
+    <h1>Our Locations</h1>
+    <ul>
+      <li>USA</li>
+      <ul>
+        <li><a href="#">New York</a></li>
+        <li><a href="#">Columbus</a></li>
+        <li><a href="#">Austin</a></li>
+        <li><a href="#">Philly</a></li>
+      </ul>
+      <li>World</li>
+      <ul>
+        <li><a href="#">Ottawa</a></li>
+        <li><a href="#">Sydney</a></li>
+      </ul>
+    </ul>
+
+    <h1>Upcoming Classes</h1>
+
+    <div class="container">
+      <div class="row justify-content-sm-left">
+        <div class="col-sm-2">Location</div>
+        <div class="col-sm-2">Topic</div>
+      </div>
+      <div class="row justify-content-sm-left">
+        <div class="col-sm-2">New York</div>
+        <div class="col-sm-2">JavaScript & jQuery</div>
+      </div>
+      <div class="row justify-content-sm-left">
+        <div class="col-sm-2">San Francisco</div>
+        <div class="col-sm-2">HTML & CSS</div>
+      </div>
+    </div>
+
+    <h1>Sign Up!</h1>
+
+    <div>
```

Comment:
Wrap everything in a `<form>` tag and add some Bootstrap style
https://getbootstrap.com/docs/4.1/components/forms/